Mathlib/Control/Writer.lean

```diff
@@ -0,0 +1,85 @@
+import Mathlib.Algebra.Group.Defs
+/-
+Copyright (c) 2019 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Simon Hudon, E.W.Ayers
+
+The writer monad transformer for passing immutable state.
+-/
```

Comment:
```suggestion
/-
Copyright (c) 2019 Microsoft Corporation. All rights reserved.
Released under Apache 2.0 license as described in the file LICENSE.
Authors: Simon Hudon, E.W.Ayers
-/
import Mathlib.Algebra.Group.Defs

/-!
The writer monad transformer for passing immutable state.
-/
```

It might be time to port the style linter. :smile: 
bors d+